relational/table.py

```diff
@@ -120,22 +120,42 @@ def _get_dbi_units(proj):
     """
     Returns:
       Net new units from DBI, only if it could be sourced from a new
-      construction permit.  None if no data from DBI.
+      construction permit or addition.  None if no data from DBI.
     """
     dbi_exist = 0
     dbi_prop = 0
     try:
-        dbi_exist = int(proj.field(
-            'existing_units', PTS.NAME,
-            entry_predicate=[_is_valid_dbi_type]))
+        fk_entries = proj.fields('existing_units',
+                                 PTS.NAME,
+                                 entry_predicate=[_is_valid_dbi_type])
```

Comment:
I think we'll want to also make sure the status isn't cancelled or withdrawn

Reply:
any suggestions on this? searching for unique statuses in the schemaless for pts shows that this field is very much not normalized. i don't *want* to do a regex on "cancel" or "withdraw" but that might be the best option otherwise

Reply:
Really, i thought when i looked at the data the 'current_status' was filled with 'cancelled' or 'withdrawn' pretty well? (Always lower case and always filled out)?

Reply:
bah you're right. my regex to go through the schemaless was bad. regex: the cause of and solution to all problems.